app/views/homes/show.html.erb

```diff
@@ -14,10 +14,43 @@
 </header>
 
 <section class="text">
-  <p>Wanna join the coolest webring? Here's how.</p>
+  <p>
+    Do you long for a simpler time, when America was Online and the only person you
+    could Ask was Jeeves? Hotline Webring is bringing that time back, with
+    Webrings!
+  </p>
+
+  <p>
+    In the 90s, search engines sucked, and people wanted a way to discover
+    related sites. &ldquo;I wish I could see more sites about puppies,&rdquo;
+    you might say to yourself over the sonorous clang of your modem, while
+    waiting for a bitmap of a Clumber Spaniel to load.
```

Comment:
I couldn't find a bitmap but look at this dork http://science-lakes.com/images/clumber-spaniel/clumber-spaniel8.jpg